src/main/java/dev/openfeature/sdk/OpenFeatureAPI.java

```diff
@@ -48,30 +48,62 @@ public static OpenFeatureAPI getInstance() {
         return SingletonHolder.INSTANCE;
     }
 
+    /**
+     * Get metadata about the default provider.
+     *
+     * @return the provider metadata
+     */
     public Metadata getProviderMetadata() {
         return getProvider().getMetadata();
     }
 
+    /**
+     * Get metadata about a registered provider using the client name.
```

Comment:
```suggestion
     * Get metadata about a registered provider using the client domain.
```

To match with domain scoping PR - https://github.com/open-feature/java-sdk/pull/934 :)

Reply:
Please see `client name` vs `domain` usage in other places

Reply:
Fixed by accepting upstream.